src/swapNumbers.js

```diff
@@ -0,0 +1,7 @@
+export default function swapNumbers(nums, number) {  
+  let numIndex = nums.indexOf(+number);
+  let nullIndex = nums.indexOf(null);
+  [nums[numIndex], nums[nullIndex]] = [nums[nullIndex], nums[numIndex]];
+  
+  return nums
+}
```

Comment:
Try to make a project structure where you would have an `utils` folder and then move all util methods like this into that folder